monero-harness/src/image.rs

```diff
@@ -200,21 +168,18 @@ pub struct MoneroWalletRpcArgs {
     pub daemon_address: String,
 }
 
-impl Default for MoneroWalletRpcArgs {
-    fn default() -> Self {
-        unimplemented!("A default instance for `MoneroWalletRpc` doesn't make sense because we always need to connect to a node.")
-    }
-}
-
 impl MoneroWalletRpcArgs {
```

Comment:
Didn't remove the debug statements or otherwise touch this one, I think that with the semi colon it doesn't return temp... Does the new() do anything in this case?